test/spa.jl

```diff
@@ -0,0 +1,25 @@
+# some tests for spa
+
+# data
+
+srand(5678)
+
+p = 5
+n = 8
+k = 3
+
+# Matrices
+
+    T=Float64
```

Comment:
Sorry for this mess. In the tests it turned out the type tuple with Float32 did not work as some internal routines blew up. The test worked for T=Float64. My reasoning was that T <: Real or Float64 should be good enough.